server/src/main/java/org/elasticsearch/cluster/SnapshotsInProgress.java

```diff
@@ -437,14 +455,21 @@ public SnapshotsInProgress(StreamInput in) throws IOException {
             if (in.getVersion().onOrAfter(REPOSITORY_ID_INTRODUCED_VERSION)) {
                 repositoryStateId = in.readLong();
             }
+            final String failure;
+            if (in.getVersion().onOrAfter(Version.V_6_7_0)) {
```

Comment:
We will have to mute BwC tests before merging this and then adjust this version to `6.7` in `master` afterwards, currently we have a `7.0.0` here in master.